site/networking.xml

```diff
@@ -0,0 +1,672 @@
+          related to TCP and IP operations, others have to do with
+          application-level protocols such as TLS. This guide covers
+          multiple topics related to networking in the context of
+          RabbitMQ. This guide is not meant to be an extensive
+          reference but rather an overview. Some tuneable parameters
+          discussed are OS-specific. This guide focuses on Linux when
+          covering OS-specific subjects, as it is the most common
+          platform RabbitMQ is deployed on in production.
+        </p>
+        <p>
+          There are several areas which can be configured or tuned:
+
+          <ul>
+            <li>Interfaces and ports</li>
+            <li>TLS</li>
+            <li>TCP socket settings</li>
+            <li>Kernel TCP settings</li>
+            <li>(AMQP 0-9-1, STOMP) Heartbeats</li>
+            <li>Hostnames and DNS</li>
+          </ul>
+
+          Except for OS kernel parameters and DNS, all RabbitMQ settings
+          are <a href="/configure.html">configured the same way</a>.
+        </p>
+        <p>
+          Networking is a broad topic. There are many configuration options
+          that can have positive or negative effect on certain workloads.
+          As such, this guide does not try to be a complete reference but rather
+          offer an index of key tunable parameters and serve as a starting
+          point.
+        </p>
+      </doc:subsection>
+    </doc:section>
+
+    <doc:section name="interfaces">
+      <doc:heading>Network Interfaces</doc:heading>
+
+      <doc:subsection name="multiple-interfaces">
+        <p>
+          For RabbitMQ to accept client connections, it needs to bind to one or more
+          interfaces and listen on (protocol-specific) ports. The interfaces are
+          configured using the <code>rabbit.tcp_listeners</code> config option.
+          By default, RabbitMQ will listen on port 5672 on all available interfaces.
+        </p>
+
+        <p>
+          TCP listeners configure both interface and port. The following example
+          demonstrates how to configure RabbitMQ on a specific IP and standard port:
+
+<pre class="code">
+[
+  {rabbit, [
+    {tcp_listeners, [{"192.168.1.99", 5672}]}
+  ]}
+].
+</pre>
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="dual-stack">
+        <doc:heading>Listening on Dual Stack (Both IPv4 and IPv6) Interfaces</doc:heading>
+        <p>
+          The following example demonstrates how to configure RabbitMQ
+          to listen on localhost only for both IPv4 and IPv6:
+
+<pre class="code">
+[
+  {rabbit, [
+    {tcp_listeners, [{"127.0.0.1", 5672},
+                     {"::1",       5672}]}
+  ]}
+].
+</pre>
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="single-stack-ipv4">
+        <doc:heading>Listening on IPv4 Interfaces Only</doc:heading>
+        <p>
+          In this example RabbitMQ will listen on an IPv4 interface only:
```

Comment:
As the guide primarily targets Linux, I would add a paragraph about the fact that this configuration on Linux (at least Debian, but probably other popular distributions) will listen to both IPv4 and IPv6. The `"127.0.0.1"` entry above is ignored by RabbitMQ on Linux because `"::1"` already takes care of it.

See: https://github.com/rabbitmq/rabbitmq-server/blob/master/src/rabbit_networking.erl#L515